tools/firmware-utils/Makefile

```diff
@@ -11,9 +11,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/firmware-utils.git
-PKG_SOURCE_DATE:=2025-01-14
-PKG_SOURCE_VERSION:=12c0b42231bee58b3cd7cbe95c33de2685484a4f
-PKG_MIRROR_HASH:=50d872a5069f9f672de5ea9efcf00872bcce27ec52bbb69fff3785abcee1c77c
+PKG_SOURCE_DATE:=2025-01-20
+PKG_SOURCE_VERSION:=2051fe5bc2a10306f0cdd8930a1cb893e433b9f4
+PKG_MIRROR_HASH:=44fd4615f2266c19be5b24651b4b6ce8913cb8c427351f5ec5ceadd959a40732
```

Comment:
This commit can be dropped now. Suppressed by: https://github.com/openwrt/openwrt/commit/1437ff840b9b81f62a7cfca02b0ba17f506ea028

Reply:
You are right, this patch will be abandoned